test/csuite/schema_abort/main.c

```diff
@@ -550,11 +564,11 @@ thread_ckpt_run(void *arg)
      */
     __wt_epoch(NULL, &start);
     for (i = 1;; ++i) {
-        sleep_time = __wt_random(&rnd) % MAX_CKPT_INVL;
+        sleep_time = __wt_random(&td->extra_rnd) % MAX_CKPT_INVL;
         sleep(sleep_time);
-        flush = ((__wt_random(&rnd) % FLUSH_INVL) == 0) ? true : false;
+        flush = ((__wt_random(&td->extra_rnd) % FLUSH_INVL) == 0) ? true : false;
         if (use_ts) {
-            ts = global_ts;
+            ts = get_all_committed_ts();
```

Comment:
This ends up different from the original intention, but this timestamp is only used for reporting progress, so it should not matter.